apps/openmw/mwsound/soundmanagerimp.cpp

```diff
@@ -109,7 +109,7 @@ namespace MWSound
 
     SoundManager::~SoundManager()
     {
-        clear();
+        SoundManager::clear();
```

Comment:
Marking the `SoundManager` class as `final` would also avoid a virtual dispatch, both here and elsewhere that it calls its own virtual methods.